code/datums/traits/negative.dm

```diff
@@ -268,3 +268,16 @@
 	var/mob/living/carbon/human/H = quirk_holder
 	phobia = new
 	H.gain_trauma(phobia, TRAUMA_RESILIENCE_SURGERY)
+
+/datum/quirk/mute
+	name = "Mute"
+	desc = "Do to some accident caused you or by choice you have no voice to be heard!"
+	value = -3 //HALP MAINTS
```

Comment:
-3 is pretty major for something that's circumvented so easily

Reply:
Just to remind that we have Pacifist on... -2, I think? Even if it's -3, I hardly think being mute is worse than being a pacifist, or even on-par with it.

Reply:
Peacefull if antaged gets removed